### PR TITLE
Update espnet api-inference-community to 0.0.32

### PR DESCRIPTION
**Pull request by StephenHodgson**
Enables proper accept header support for audio based tasks using espnet

Continued from #287

### CODE REVIEW

File: docker_images/espnet/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-api-inference-community==0.0.23
+api-inference-community==0.0.35
```
Comment (StephenHodgson) [inline docker_images/espnet/requirements.txt:1]:
```suggestion
api-inference-community==0.0.32
```

**Comment by Vaibhavs10** [comment]
Hey @StephenHodgson - Thank you for your contribution! I was double-checking the tests for the current changes.
The changes break the Python environment for me:
I had to change the version of `pyopenjtalk` and `torchaudio` to be able to build the environment again.

Even after that, only 3 out of 9 tests `Passed`.

```
docker_images/espnet/tests/test_api.py::PipelineTestCase::test_has_at_least_one_task_enabled PASSED
docker_images/espnet/tests/test_api.py::PipelineTestCase::test_unsupported_tasks PASSED
docker_images/espnet/tests/test_api_automatic_speech_recognition.py::AutomaticSpeecRecognitionTestCase::test_duDownloading: 100%|████████████████████████████████████████████████████████████| 690/690 [00:00<00:00, 4.27MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 1.95k/1.95k [00:00<00:00, 10.2MB/s]
Downloading: 100%|██████████████████████████████████████████████████████████| 238k/238k [00:00<00:00, 5.28MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 1.40k/1.40k [00:00<00:00, 9.82MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 33.1M/33.1M [00:00<00:00, 77.2MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 1.69k/1.69k [00:00<00:00, 7.92MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 2.78k/2.78k [00:00<00:00, 15.7MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 27.2M/27.2M [00:00<00:00, 45.5MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 2.09k/2.09k [00:00<00:00, 11.8MB/s]
Downloading: 100%|████████████████████████████████████████████████████████████| 330/330 [00:00<00:00, 2.00MB/s]
FAILED
docker_images/espnet/tests/test_api_automatic_speech_recognition.py::AutomaticSpeecRecognitionTestCase::test_malformed_audio FAILED
docker_images/espnet/tests/test_api_automatic_speech_recognition.py::AutomaticSpeecRecognitionTestCase::test_original_audiofile FAILED
docker_images/espnet/tests/test_api_automatic_speech_recognition.py::AutomaticSpeecRecognitionTestCase::test_webm_audiofile FAILED
Downloading: 100%|████████████████████████████████████████████████████████████| 690/690 [00:00<00:00, 3.14MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 1.91k/1.91k [00:00<00:00, 13.1MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 6.29k/6.29k [00:00<00:00, 24.2MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 30.4k/30.4k [00:00<00:00, 30.5MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 33.6k/33.6k [00:00<00:00, 20.2MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 25.7k/25.7k [00:00<00:00, 28.5MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 38.6k/38.6k [00:00<00:00, 21.3MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 26.1k/26.1k [00:00<00:00, 25.4MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 30.0k/30.0k [00:00<00:00, 31.8MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 23.9k/23.9k [00:00<00:00, 54.6MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 28.4k/28.4k [00:00<00:00, 29.7MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 23.0k/23.0k [00:00<00:00, 51.2MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 35.2k/35.2k [00:00<00:00, 3.71MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 75.4k/75.4k [00:00<00:00, 3.01MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 25.3k/25.3k [00:00<00:00, 24.0MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 25.9k/25.9k [00:00<00:00, 25.3MB/s]
Downloading: 100%|██████████████████████████████████████████████████████████| 149M/149M [00:01<00:00, 87.4MB/s]
Downloading: 100%|████████████████████████████████████████████████████████████| 770/770 [00:00<00:00, 5.97MB/s]
Downloading: 100%|████████████████████████████████████████████████████████| 1.40k/1.40k [00:00<00:00, 9.47MB/s]
Downloading: 100%|████████████████████████████████████████████████████████████| 770/770 [00:00<00:00, 6.16MB/s]
Downloading: 100%|████████████████████████████████████████████████████████████| 323/323 [00:00<00:00, 2.38MB/s]
PASSED
docker_images/espnet/tests/test_api_text_to_speech.py::TextToSpeechTestCase::test_simple FAILED
docker_images/espnet/tests/test_docker_build.py::DockerBuildTestCase::test_can_build_docker_image Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/build?buildargs=%7B%7D&cachefrom=%5B%5D&cgroupparent=&cpuperiod=0&cpuquota=0&cpusetcpus=&cpusetmems=&cpushares=0&dockerfile=Dockerfile&labels=%7B%7D&memory=0&memswap=0&networkmode=default&rm=1&shmsize=0&target=&ulimits=null&version=1": dial unix /var/run/docker.sock: connect: permission denied
FAILED
```

Can you please look into this and adjust the requirements accordingly!

**Comment by Vaibhavs10** [comment]
So upon some more playing around, I found that the version of `numpy` is the offending issue. We definitely need to work on the `requirements` such that the environment resolves.

**Comment by osanseviero** [comment]
@Vaibhavs10 could you share more details, please? Which `numpy` version do you have? Which version fixed the issue? FYI we already have `numpy` in one of the dependencies (`api-inference-community`, see [here](https://github.com/huggingface/api-inference-community/blob/main/requirements.txt#L2)); are you sure you had installed all the dependencies in a clean environment and you have a `numpy` version that meet the requirements?

**Comment by Vaibhavs10** [comment]
Here's the repro (@osanseviero):

1. `python -m venv test_env`
2. `source test_env/bin/activate`
3.  `pip install -r docker_images/espnet/requirements.txt`

It throws an error while installing `pyopenjtalk` - it refuses to build.

Then, I upgrade `pyopenjtalk` to the latest version (0.3.2)

Next, pip throws an error for `torchaudio` - No matching versions found for `0.8.1`

Then, I upgraded it to the latest version (2.0.2)

After all the above steps, the environment is resolved and created.

However, after this if you run the tests for ESPNet then ASR and TTS tests fail. Upon prodding further I received the error w.r.t. the Numpy version, as internally ESPNet casts using `np.int` this has been deprecated in the recent version.

Tested it on a Linux VM.

**Pull request closed without merging**